libraries/matrixui/src/main/kotlin/io/element/android/libraries/matrix/ui/media/ImageLoaderFactories.kt

```diff
@@ -46,8 +46,8 @@ class LoggedInImageLoaderFactory @Inject constructor(
                 }
                 add(AvatarDataKeyer())
                 add(MediaRequestDataKeyer())
-                add(CoilMediaFetcher.AvatarFactory(matrixClient))
-                add(CoilMediaFetcher.MediaRequestDataFactory(matrixClient))
+                add(CoilMediaFetcher.AvatarFactory(context, matrixClient))
+                add(CoilMediaFetcher.MediaRequestDataFactory(context, matrixClient))
```

Comment:
Not a blocker, but instead of passing the whole `context`, can we just pass the screen density here?
Not sure that this can change over time, but maybe (with battery saver mode which simulate low density screen).

Or maybe just `context.resources.displayMetrics`.

I am definitely thinking out loud ;)

Reply:
I'd have to double check, because the class has an `@Inject` annotation. So I think we need to either remove it (not sure if it's possible) or we manually provide the density value it in a module function.

Reply:
Actually, I'm wondering about what would happen if we passed it here and then changed the settings for display element scaling in the OS. Maybe we should pass a lambda or a wrapper `DensityProvider` class that hides the context?

Reply:
Done in https://github.com/vector-im/element-x-android/pull/1262/commits/e7e41b058938e0de08a4beb5b10250cde8ca0858.